packages/remark-ping-zds/src/index.js

```diff
@@ -0,0 +1,63 @@
+const request = require('sync-request')
+
+const pattern = /[\s'"(,:<]?@(?:\*\*([^*]+)\*\*|(\w+))/
+
+function locator (value, fromIndex) {
+  const keep = pattern.exec(value, fromIndex)
+  if (keep) {
+    return value.indexOf('@', keep.index)
+  }
+  return -1
+}
+
+function plugin () {
+  function zdsMemberExists (username) {
+    //TODO https://github.com/zestedesavoir/zds-site/issues/4382
+    let isMember = false
+    const res = request('GET', `http://zestedesavoir.com/api/membres/?search=${username}`)
```

Comment:
Hmmm, I'm not sure actually. It's a very specific plugin for a specific case. If anyone want to use this plugin for another website, they must provide an endpoint which return a code 200 if the user exists.